packages/gafl-webapp-service/src/pages/order-complete/licence-information/licence-information.njk

```diff
@@ -6,7 +6,7 @@
 {% from "summary-list/macro.njk" import govukSummaryList %}
 {% from "back-link/macro.njk" import govukBackLink %}
 
-{% set title = "Licence number " + data.permission.referenceNumber %}
+{% set title = "Licence Information " %}
 {% block pageTitle %}{{ title }} - GOV.UK{% endblock %}
```

Comment:
You've got a double space after 'Information'

Reply:
oh goodness, good spot